modules/services/fb_ai_features.js

```diff
@@ -64,7 +107,11 @@ function getTags(obj) {
     var tags = {};
     for (var i = 0, l = elems.length; i < l; i++) {
         var attrs = elems[i].attributes;
-        tags[attrs.k.value] = attrs.v.value;
+        var k = (attrs.k.value || '').trim();
```

Comment:
Totally not a big deal, but were there instances of tag keys/values actually containing trailing whitespace? I'd have thought trailing whitespace wouldn't be transmitted over the wire.